DDCore/src/segmentations/HexGrid.cpp

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
License notice missing

Reply:
Added license info in [ade020d](https://github.com/sebouh137/DD4hep/commit/ade020d946c0fab99b8266a4a7823d225f314255)